stage1/modules/node-materials/node/projects/notes.md

```diff
@@ -113,11 +112,13 @@ function list() {
 }
 ```
 
-Так как файл `notes.json` лежит в той же директории, что и файл с кодом, можно не прописывать к нему путь, достаточно указать его название первым аргументов метода `readFile()`. Второй аргумент метода - функция обратного вызова, которая принимает два параметра - ошибку `error` и прочитанные из файла данные `data`.  
-Преобразовать полученные данные в массив позволяет метод `JSON.parse(data)`.  
-Идём по этому массиву при помощи метода `forEach()` и для каждого элемента выводим в консоль его индекс и название заметки, так что в консоль выводится нумерованный список. Чтобы список начинался не с нуля, а с единицы, к индексу прибавим 1.
+Since the `notes.json` file is in the same directory as the code file, there is no need to specify the path; it's enough to provide its name as the first argument to the `readFile()` method. The second argument of the method is a callback function that takes two parameters - an error (`error`) and the data read from the file (`data`).
```

Comment:
```suggestion
Since the `notes.json` file is located in the same directory as the code file, there's no need to specify its full path. It's enough to provide its name as the first argument to the `readFile()` method. The second argument of this method is a callback function, which accepts two parameters: an `error` for any encountered errors, and the `data` read from the file.
```